.github/workflows/appium-automation.yml

```diff
@@ -20,12 +21,20 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: sdk
+          ref: ${{ github.ref }}
       # point to automation master branch by default
+      - name: Checkout Master SDK repo with testbed app for Reinstallation tests
+        uses: actions/checkout@v3
+        if: ${{ !( github.ref == 'refs/heads/master')}}
+        with:
+          path: previousVersionSDK
+          ref: Test-Automation-Improvements
       - name: Checkout qa automation
         uses: actions/checkout@v3
         with:
           path: qa
           repository: BranchMetrics/qentelli-saas-sdk-testing-automation
+          ref: Improvements
```

Comment:
Approved automation side, update before merge

Reply:
@gdeluna-branch Done. Please re-approve it.